### DISCUSSION TITLE
query

### DISCUSSION THREAD

**Discussion by Fatimaijax** [AammarTufail/python_ka_chilla2023#118] [Q&A]
hi, sir why you don't use Jupiter notebook for python coding?

**Comment by SahilSS123** [comment]
> hi, sir why you don't use Jupiter notebook for python coding?

 Vs code is very  easy to use and this provide very easy to use environment

**Comment by kousarraza** [comment]
Jupiter notebook is used  for advance level such as ML, DL ,AI coding. For Simple/ Intermediate Python VS code is best.
